core/src/wallet/bitcoin/database/BitcoinLikeTransactionDatabaseHelper.cpp

```diff
@@ -55,13 +55,13 @@ namespace ledger {
         }
 
         std::string BitcoinLikeTransactionDatabaseHelper::putTransaction(soci::session &sql,
-                                                                  const std::string& accountUid,
-                                                                  const BitcoinLikeBlockchainExplorerTransaction &tx) {
+                                                                         const std::shared_ptr<AbstractAccount> &account,
```

Comment:
[OPT][FAR] With this change we coupling DB level and logic level more. Ideally TransactionDatabaseHelper should not know about the keychain logic.